jsonschema-module-jakarta-validation/README.md

```diff
@@ -13,6 +13,7 @@ Module for the [jsonschema-generator](../jsonschema-generator) – deriving JSON
 7. Populate "pattern" for strings. Based on `@Pattern`/`@Email`, when corresponding `JakartaValidationOption.INCLUDE_PATTERN_EXPRESSIONS` is being provided in constructor.
 8. Populate "minimum"/"exclusiveMinimum" for numbers. Based on `@Min`/`@DecimalMin`/`@Positive`/`@PositiveOrZero`.
 9. Populate "maximum"/"exclusiveMaximum" for numbers. Based on `@Max`/`@DecimalMax`/`@Negative`/`@NegativeOrZero`.
+10. Populate "enum"/"const" for booleans. Based on `@AssertTrue`/`@AssertFalse`.
```

Comment:
Thanks for adding this to the `README` straightaway.
Should also add this to the [corresponding documentation](https://github.com/victools/jsonschema-generator/blob/main/slate-docs/source/includes/_jakarta-validation-module.md) at the same time.

But I can take care of that afterward, same as the [`CHANGELOG`](https://github.com/victools/jsonschema-generator/blob/main/CHANGELOG.md) and the list of contributors in the parent [`pom.xml`](https://github.com/victools/jsonschema-generator/blob/main/jsonschema-generator-parent/pom.xml).

Reply:
I updated the slate-docs. I'll leave the changelog and contributors to you.